dotCMS/src/main/java/com/dotcms/timemachine/business/TimeMachineAPIImpl.java

```diff
@@ -197,4 +203,32 @@ public boolean accept(File dir, String name) {
 		}
     }
 
+    @Override
+    public List<File> removeOldTimeMachineBackupsFiles() {
+        long now = Instant.now().toEpochMilli();
+        final List<File> fileToRemove = new ArrayList<>();
+        final File timeMachinePath = new File(ConfigUtils.getTimeMachinePath());
+
+        for ( File file : timeMachinePath.listFiles()) {
+            if ( file.isDirectory() && file.getName().startsWith("timeMachineBundle_")) {
+
+                long lastModifiedDays = TimeUnit.MILLISECONDS.toDays(now - file.lastModified());
+
+                if (lastModifiedDays > PRUNE_TIMEMACHINE_OLDER_THAN_DAYS.get()) {
+                    fileToRemove.add(file);
+                }
+            }
+        }
+
+        for (File file : fileToRemove) {
+            try {
+                FileUtil.deleteDir(file.toPath().toString());
+            } catch (IOException e) {
+                //ignore
```

Comment:
We should log something here indicating the file that was unable to being removed

Reply:
agree

Reply:
done https://github.com/dotCMS/core/pull/28530/commits/6e6757d800ae36575a4fd3854c1a6e164d8bd327#diff-19b28e2982b6db869240fb8fc96a2bf2c156e3607f41656fd16d7845c45ba604R228